Pa3/Durchführung/Project/app/StateHandler.c

```diff
@@ -9,19 +9,19 @@
 /**************************************************************************************************/
 
 /* INCLUDES ***************************************************************************************/
-#include "StateHandler.h"
-
-#include "EventEnum.h"
-#include "InitializeMcu.h"
-#include "CalibrateLineSensors.h"
-#include "ReadyToDrive.h"
-#include "DriveToStart.h"
-#include "DriveToFinish.h"
-#include "DriveOverGap.h"
-#include "DisplayCountdown.h"
-#include "DisplayLapTime.h"
-#include "ErrorHandlerState.h"
-#include "SetParameters.h"
+#include "app/StateHandler.h"
+
+#include "app/EventEnum.h"
```

Comment:
full path should not be needed if files are in the same folder
```suggestion
#include "StateHandler.h"

#include "EventEnum.h"
```